docs/fundamentals/jcloud/yaml-spec.md

```diff
@@ -352,26 +352,14 @@ Keys in `labels` have the following restrictions.
     - retention-days
 ```
 
-
+(jcloud-lifetime)=
 ### Lifetime
 
 A Flow that receives no traffic in 24 hours will be automatically deleted by default.
 
-To ignore the lifetime reclaim policy of a Flow, you can use the `retention_days` parameter in the Flow yaml. `retention_days` will keep the flow alive for `x` days (0<x<365). flows is going to be removed after `x` days regardless of above reclaim policy. `-1` is to keep the flow alive regardless of the reclaim policy.
-
-```{note}
-- If {ref}`retention-days <retention-days>` argument configured as `x` (0<x<365). Flows will be removed after `retention-days`, regradless of the usage.
-
-- If {ref}`retention-days <retention-days>` argument configured as `-1`. Flows will not be removed, regradless of the usage.
+- Maximal retention days for CPU Flows is 3 days
+- Maximal retention days for GPU Flows is 0.5 days.
```

Comment:
We should still include how you can precise a retention days value. I would suggest to add the same as before but just precise that there is a limit of 3/0.5 days

Please put the yaml back

Reply:
So then (0<x<365) should be 0<x<3? Having it as before could be misleading

Reply:
Yes exactly